kiosk/src/Models/Kiosk.ts

```diff
@@ -327,13 +327,14 @@ export class Kiosk {
 
         function createIFrame(src: string) {
             const iframe: HTMLIFrameElement = document.createElement("iframe");
-            iframe.id = "sim-embed";
+            iframe.className = "sim-embed";
             iframe.frameBorder = "0";
             iframe.setAttribute("sandbox", "allow-popups allow-forms allow-scripts allow-same-origin");
             iframe.src = src;
             return iframe;
         }
         gamespace.appendChild(createIFrame(playUrlBase + playQueryParam)).focus();
+        console.log(document.activeElement);
```

Comment:
Did you mean to leave this console log in?

Reply:
Gosh dang it, no. Thanks. Taking it out.